projects/sbb-esta/angular-maps/esri-web-map/src/esri-web-map/esri-web-map.component.spec.ts

```diff
@@ -14,7 +13,7 @@ describe('EsriWebMapComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [EsriWebMapComponent],
-      providers: [EsriTypesService, HitTestService, GraphicService, GraphicTypes]
+      providers: [EsriTypesService, HitTestService, GraphicService]
```

Comment:
Services provided in 'root' usually do not need to be added to providers in tests, unless you want to mock them.